lib/assessment.js

```diff
@@ -32,14 +36,20 @@ var any = function any(array, callback, thisArg) {
 // made up of the array elements for which invoking
 // the callback returns a truthy value
 var select = function select(array, callback, thisArg) {
-
+  newArray = [];
+  if (array.forEach(callback) === true) {
+    newArray = array.map(callback);
+  }
+  return newArray;
 };
 
 // Question 5
 // Write a function that returns the first element
 // for which invoking the callback returns a truthy value
 var find = function find(array, callback, thisArg) {
-
+  if (!array.every(callback) === false) {
+    return array[i];
+  }
```

Comment:
Nope, sorry. As above, let's consider a high-level solution in english.

In order to return the first element which matches our condition (i.e. causes our callback to return `true` when passed in as an argument), we must iterate through every element in the array, checking each element against our callback; if we find a match, we end early and return the matching value.

Does that give you any clues as to how to solve this one?